docs/channels/emails.rst

```diff
@@ -225,15 +225,53 @@ Marketers can place the signature into an Email using the ``{signature}`` token.
 
-- :xref:`Mandrill`
-- :xref:`GMail`
-- :xref:`Sendgrid`
-- :xref:`Mailjet`
-- :xref:`Postmark`
-- :xref:`Sendmail`
-- :xref:`Amazon SES`.
+SMTP Transport
+**************
+
+The SMTP transport is the default transport for Mautic. It's configured in the Mautic configuration under the Email Settings tab. The configuration is the same as in the :xref:`Symfony Mailer` documentation.
+
+This is the example DSN mentioned in the :xref:`Symfony Mailer` documentation for SMTP:
+
+.. code-block:: shell
+    
+    smtp://user:pass@smtp.example.com:port
+
+This is how this DSN converted to Mautic's email configuration:
+
+.. image:: images/emails/smtp-dsn.png
+    :width: 400
+    :alt: SMTP API DSN example
+
+.. note::
+  Use the Mautic's global configuration to paste in the DSN information. Especially the API keys and passwords. The values must be URL-encoded and the configuration form will do that for you. If you are pasting DSN directly into the config/local.php file, you must URL-encode the values yourself.
+
+Example API Transport Installation
+**********************************
+
+Let's say you want to use :xref:`Sendgrid` API instead of SMTP to send Emails. You can install the official Symfony Sendgrid Transport by running the following command that is mentioned along others in the :xref:`Symfony Mailer` documentation.
+
+.. code-block:: shell
+    
+    composer require symfony/sendgrid-mailer
+
+After that, you can configure the transport in the Mautic configuration. The example DSN is again mentioned in the :xref:`Symfony Mailer` documentation along with other transports. In case of Sendgrid API, the DSN would look like this:
+
+.. code-block:: shell
+    
+    sendgrid+api://KEY@default
+
+This is how this DSN converted to Mautic's email configuration:
+
+  .. image:: images/emails/sendgrid-api-dsn.png
+    :width: 400
+    :alt: Sendgrid API DSN example
+
+Once you replace the Sendgrid API KEY, you can add it to the DSN and save the configuration. Mautic will now use the Sendgrid API to send Emails. 
+
+.. note::
+  It is nice perk that Mautic can use any Symfony Mailer transport. However, be aware that such transport do not support batch sending even via API. They only send one email per request opposed to a thousand emails per request as some special Mautic tansports can. It also does not support transport callback handling. If you plan to send bigger ammounts of emails please consider using email transports built specifically for such use. These plugins are available in :doc:`Mautic Marketplace </marketplace/marketplace.rst>`.
 
 The system can either send Emails immediately or queue them for processing in batches by a :doc:`cron job </configuration/cron_jobs>`.
```

Comment:
I think we should kill the immediate mode from the docs and ask users to use the queue mode all the time

Reply:
The immediate mode is using `sync://` messenger option, right? Or are you talking about something else?

Reply:
Yes, that is right. I suggest we do not use sync at all

Reply:
I thin it's useful when someone is just testing Mautic out or have Mautic with a few dozens of contacts. @RCheesley WDYT?

Reply:
I do think that immediate mode has a place for low-volume sending or when the mails really do need to go out immediately.

I'd only be in favour of killing it if we're absolutely sure that every single user of Mautic on any hosting environment is going to be capable and willing to use the queueing system, and we can document how to make that effectively send the emails as if they were immediate.